megamek/src/megamek/client/ui/swing/TeleMissileSettingDialog.java

```diff
@@ -42,10 +44,15 @@
     private JTextField fldSetting = new JTextField("50", 2); //$NON-NLS-1$
     private int setting;
     private JFrame frame;
+    private int minimumVelocity = 1;
+    private int maxVelocity = 50;
```

Comment:
Is this 50 or 500? Previous place showed 500. Could these be added as statics to a class?

Reply:
I couldn't figure out what class to put them under and I only use the data here so a public static int seemed a little silly, but was trying to avoid the use of 'magic numbers' here.
50 is the RAW value, 500 is the one I made up for this option.  (Nobody really wants to launch a missile at a target 5500 hexes away and wait 100 rounds...)
I stuck myself with an NPE when I tried to declare the variable value by pulling directly from the game option setting, so I did it this way.

Reply:
Fixed now.  Amazing what a fresh cup of coffee will do.